sections/main-article.liquid

```diff
@@ -274,14 +274,6 @@
     },
     "publisher": {
       "@type": "Organization",
-      {% if settings.share_image %}
-        "logo": {
-          "@type": "ImageObject",
-          "height": {{ settings.share_image.height | json }},
-          "url": {{ settings.share_image | image_url: width: settings.share_image.width | prepend: "https:" | json }},
-          "width": {{ settings.share_image.width | json }}
-        },
-      {% endif %}
```

Comment:
Removed this as it's not outputting ever as a `share_image` setting doesn't exists.